### DISCUSSION TITLE
CacheManager.StackExchange.Redis add list type data problem

### DISCUSSION THREAD

**Discussion by landonzeng** [MichaCo/CacheManager#350] [Q&A] [answered]
I'm using CacheManager.StackExchange.Redis in my NetCore 3.1 project, and when I added a cache, I deposited a list, but when I checked it with RDM, I found that it was a HASH type, not a list type, so how do I add a list to it, because I want to add a list in I want to add data or modify data, I just need to update the object corresponding to the list, not to read out all the data of the key, then operate on the list, and then stuff it into the key, I am worried that the performance loss will be large, what is a good solution?

**Comment by MichaConrad** [comment, marked as answer]
All Redis keys CacheManager creates are hash keys because I need a couple properties as meta data. That's a tradeoff so far.
If you really have to update a list in Redis all the time and that's all you do, then you are probably better off by just using Redis directly/only
